tasks/main.yml

```diff
@@ -18,5 +18,7 @@
     name: "{{ item.path }}"
     opts: "{{ item.opts | default(nfs_mount_opts) }}"
     src: "{{ item.location }}"
+    dump: "{{ item.dump | default(omit) }}"
+    passno: "{{ item.passno | default(omit) }}"
     state: mounted
```

Comment:
Would the defaults for both not be `0` or is that the effective behaviour of `omit`?

Reply:
`omit` means default to whatever the ansible role encodes as the default

Reply:
Not exactly (as best I can tell). I needed to add `0` for things to work as expected. I don't understand or particularly like the default behavior but to get this in as a minor release, I thought this was the best change.